framework/config/loader.go

```diff
@@ -215,8 +214,14 @@ func loadYamlConfig(area *Area, config []byte) error {
 
 func loadYamlRoutesFile(area *Area, filename string) error {
 	routes, err := ioutil.ReadFile(filename + ".yml")
-	if err != nil {
-		return err
+	if err == nil {
+		return yaml.Unmarshal(routes, &area.Routes)
 	}
-	return yaml.Unmarshal(routes, &area.Routes)
+
+	routes, err = ioutil.ReadFile(filename + ".yaml")
+	if err == nil {
+		return yaml.Unmarshal(routes, &area.Routes)
+	}
+
+	return err
```

Comment:
should we return a more specific error message?
It can be confusing if I want to load a `.yml` and only find the error message of the last `.yaml` ReadFile